src/note/migrations/0004_notepriority.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10 on 2017-01-15 04:19
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def set_initial_priorities(apps, schema_editor):
+    # set NotePriority initial
+    NotePriority = apps.get_model('note', 'NotePriority')
+    NotePriority.objects.create(name="Normal")
+    NotePriority.objects.create(name="Urgent")
```

Comment:
While this PR is not merged, `Normal` and `Urgent` are values manually created in the python code.  After this PR this values become into a Django model. I respected the original values in the migration files. 
Now, there are an important thing, we probably need a model translation solution for the nomenclatures as (`NotePriority` and `NoteCategory`)